config/daemonconfig.go

```diff
@@ -61,8 +61,12 @@ type DaemonConfig struct {
 }
 
 type MirrorConfig struct {
-	Host    string            `json:"host,omitempty"`
-	Headers map[string]string `json:"headers,omitempty"`
+	Host                string            `json:"host,omitempty"`
+	Headers             map[string]string `json:"headers,omitempty"`
+	AuthThrough         bool              `json:"auth_through,omitempty"`
+	HealthCheckInterval int               `json:"health_check_interval,omitempty"`
+	FailureLimit        uint8             `json:"failure_limit,omitempty"`
```

Comment:
Why HealthCheckInterval uses  `uint8` but HealthCheckInterval uses `int`?

Reply:
Sorry, the `HealthCheckInterval` is refer to `CheckInterval` for `proxy`. Should we use `int` for both of them?

Reply:
I also find other items which have different types between `nydus` and `nydus-snapshotter`. Should we fix it together?

nydus:
```rust
pub timeout: u32,
pub connect_timeout: u32,
pub retry_limit: u8,
```

nydus-snapshotter:
```golang
Timeout        int `json:"timeout,omitempty"`
ConnectTimeout int `json:"connect_timeout,omitempty"`
RetryLimit     int `json:"retry_limit,omitempty"`
```

Reply:
And the `FailureLimit ` references to the `RetryLimit` in `proxy`.

Reply:
@sctb512 Thanks for explaining, it's ok, we can make snapshotter consistent with nydusd.